Makefile

```diff
@@ -32,6 +35,12 @@ test:
 	go vet -mod=readonly  ./...
 	go test -mod=readonly -race ./...
 
+lint:
+ifndef LINT
+	./scripts/golangci-lint.sh -b $(shell go env GOPATH)/bin v1.17.1
```

Comment:
I just thought, is there a docker container available, that we could use instead?
We are using a docker container to compile protocol buffer files and it works great.

Reply:
That is a good call for a future PR, I'd like to do that for sure. Saves us from the necessity of having to download 3rd party stuff from github.

Reply:
Actually, I would love to see this in a Docker image. It makes all tools much simpler.

After looking at that code, I don't see why not. Even a Dockerfile in this repo.
I will try that

Reply:
Or what about https://hub.docker.com/r/golangci/golangci-lint/tags ?

Reply:
I think it is fair to fit that in this PR